#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_fix_entropy.dpatch by Stefan <stefan.esser@sektioneins.de>
##
## DP: Fix entropy (http://bugs.debian.org/539307)

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' php-suhosin-0.9.31~/execute.c php-suhosin-0.9.31/execute.c
--- php-suhosin-0.9.31~/execute.c	2010-04-13 11:36:29.000000000 +0200
+++ php-suhosin-0.9.31/execute.c	2010-04-13 11:37:07.000000000 +0200
@@ -28,6 +28,7 @@
 #include "zend_hash.h"
 #include "zend_extensions.h"
 #include "ext/standard/info.h"
+#include "ext/standard/php_lcg.h"
 #include "ext/standard/php_rand.h"
 #include "php_suhosin.h"
 #include "zend_compile.h"
